### ISSUE TITLE
AirPlay not working

### ISSUE THREAD

**Issue by inickt** [inickt/LazyMan-iOS#35]

**Comment by inickt** [comment]
Seems AVAssetResourceDelegate is not getting called for AirPlay. Not sure if this can be resolved.

**Comment by DKDigitalMedia** [comment]
Ah okay, was wondering why it wouldn't play on my AppleTV...  :)

**Comment by nahart86** [comment]
I can get AirPlay to work with iOS 12.0 and AppleTV version 12.0 by air playing my iPhone screen to the TV, then launching the LazyMan app and displaying the video and finally tapping the full-screen button. However, as the issue states, I cannot begin the video from Lazyman and press the airplay button.

**Comment by inickt** [comment]
@nahart86 yeah that should work because it is just showing your screen. I don't think it will actually be possible to use AirPlay the normal way with the app just because of the way we have to handle the streams.

**Comment by inickt** [comment]
It's not possible to cast your screen as far as I am aware.
